bundles/configuration.rst

```diff
@@ -269,9 +269,9 @@ In your extension, you can load this and dynamically set its arguments::
         $configuration = new Configuration();
         $config = $this->processConfiguration($configuration, $configs);
 
-        $def = $container->getDefinition('acme.social.twitter_client');
-        $def->replaceArgument(0, $config['twitter']['client_id']);
-        $def->replaceArgument(1, $config['twitter']['client_secret']);
+        $twitterClient = $container->getDefinition('acme.social.twitter_client');
```

Comment:
`$twitterClientDefinition`. I remember getting confused the first time I came across definitions because of a similar naming as it was named as if it was the service but was not the service

Reply:
I would rather revert it to `$definition` I saw some places where that name is used.

Reply:
true, unless there is multiple ones you might not care about a better naming